Portuguese (Brazil)

KillerJoeBR opened the pull request:
First time doing this, so please forgive if i'm doing it wrong. Here is the translation for Portuguese (Brazil)

[Portuguese (Brazil).zip](https://github.com/steeve/plugin.video.pulsar/files/113274/Portuguese.Brazil.zip)

scakemyer committed:
Update to v0.9.18

m4xc4v413r4 committed:
Updated Portuguese Translation

Updated to v0.9.18

scakemyer committed:
Merge pull request #224 from m4xc4v413r4/patch-2

Updated Portuguese Translation

scakemyer committed:
add TheMovieDB API key setting, #226

scakemyer committed:
add multi-cloud icon support, #171

scakemyer committed:
Update to v0.9.19

assapir committed:
Update Hebrew

scakemyer committed:
Merge pull request #227 from meijin007/master

Update Hebrew

zorg1977 committed:
New translations/fix existing in French strings.po

zorg1977 committed:
Fix forum URL

scakemyer committed:
Merge pull request #232 from zorg1977/patch-3

Fix forum URL

scakemyer committed:
Merge pull request #231 from zorg1977/patch-2

New translations/fix existing in French strings.po

scakemyer committed:
Update to v0.9.20

m4xc4v413r4 committed:
Updated Portuguese Translation

Updated for v0.9.20

scakemyer committed:
Merge pull request #235 from m4xc4v413r4/patch-4

Updated Portuguese Translation

jellespijker committed:
Updated Dutch translations

scakemyer committed:
Merge pull request #236 from peer23peer/master

Updated Dutch translations

penzinho committed:
Updated Croatian translation up to v.0.9.20

scakemyer committed:
Merge pull request #238 from penzinho/master

Updated Croatian translation up to v.0.9.20

lrrosa committed:
Update Portuguese (Brazil) translation

Added TheMovieDB API key

Filosofos committed:
Greek translation update

scakemyer committed:
Merge pull request #242 from Filosofos/patch-3

Greek translation update

scakemyer committed:
Merge pull request #241 from lrrosa/patch-2

Update Portuguese (Brazil) translation

scakemyer committed:
Trakt icon and public lists, #31

scakemyer committed:
add Download path not set message

scakemyer committed:
Update to v0.9.21

penzinho committed:
Croatian translation up to date with v.0.9.21

scakemyer committed:
Merge pull request #244 from penzinho/master

Croatian translation up to date with v.0.9.21

zorg1977 committed:
French strings.po updated to reflect 0.9.21

NOTE: Wasn't easy to translate Trakt concepts. Maybe be subject to future adjustments

Greenbaburu committed:
Update italian translation #30040 / 30113 / 30251

scakemyer committed:
Merge pull request #247 from zorg1977/patch-4

French strings.po updated to reflect 0.9.21

scakemyer committed:
Merge pull request #250 from Greenbaburu/patch-8

Update italian translation #30040 / 30113 / 30251

lrrosa committed:
Update Portuguese (Brazil) translation

Reflects changes in 0.9.21

scakemyer committed:
Merge pull request #251 from lrrosa/patch-3

Update Portuguese (Brazil) translation

assapir committed:
Hebrew update

scakemyer committed:
add resolution preference settings, #191

scakemyer committed:
catch SystemExit exceptions, possible fix for #214

scakemyer committed:
Update to v0.9.22

scakemyer committed:
Merge pull request #254 from meijin007/master

Hebrew update

abuisine committed:
French translation updated

scakemyer committed:
Merge pull request #258 from abuisine/translation_french

French translation updated

Commit:
Updated Portuguese to latest version and added the stuff @zorg1977 was adding in #248

scakemyer committed:
Merge pull request #260 from m4xc4v413r4/master

Updated Portuguese Translations and some extra stuff

scakemyer committed:
pass a few more infolabels to VideoPlayer, #256

scakemyer committed:
support for search_season method, #195

penzinho committed:
Croatian translation updated v.0.9.22

johnnyslt committed:
Updated Romanian translations

zorg1977 committed:
Updated French translations

A few fixes
A proposition regarding Trakt submenus: items could be prefixed with 'Trakt:' for the user to know where do these lists come from

assapir committed:
Update Hebrew

scakemyer committed:
Merge pull request #276 from meijin007/master

Update Hebrew

scakemyer committed:
Merge pull request #274 from zorg1977/patch-5

Updated French translations

scakemyer committed:
Merge pull request #272 from johnnyslt/master

Updated Romanian translations

scakemyer committed:
Merge pull request #270 from penzinho/master

Croatian translation updated v.0.9.22

KillerJoeBR committed:
Updated Portuguese (Brazil) translation

Added new messages to the translation

scakemyer committed:
Merge pull request #278 from KillerJoeBR/patch-7

Updated Portuguese (Brazil) translation

scakemyer committed:
Update to v0.9.23

Filosofos committed:
Updated Greek translation

Version 0.9.23

scakemyer committed:
Merge pull request #284 from Filosofos/patch-7

Updated Greek translation

a1ex4 committed:
More infolabels

scakemyer committed:
Merge pull request #287 from likeitneverwentaway/quasar-my-branch

More infolabels,#256

Greenbaburu committed:
Update italian translation #30049 / 30113 / 30251

scakemyer committed:
Merge pull request #291 from Greenbaburu/patch-9

Update italian translation #30049 / 30113 / 30251

razer1s committed:
Updated Russian translation

added new sting

scakemyer committed:
Merge pull request #298 from razer1s/patch-5

Updated Russian translation

abuisine committed:
new icons (try #2) to replace redundant trakt icon

replaces PR#283 which had bad alpha
the “organiser” icon was smaller to compensate that it is fat compared
to the others
it is now available with the same size

scakemyer committed:
Merge pull request #306 from abuisine/new_icons#2

new icons (try #2) to replace redundant trakt icon

Commit:
Added German Translation

jellespijker committed:
Dutch translations updated

Albertron86 commented:
Hi, i use your plugin in Windows kodi and its perfect. Also I have a receiver vu+ SOLO2 based on Linux , but does not work any press plugin. Mipsel can be implemented in git , then to compile the addon? It would be great

scakemyer committed:
Merge pull request #307 from eidermar/master

Added German Translation

scakemyer committed:
Merge pull request #319 from peer23peer/master

Dutch translations updated

scakemyer committed:
add linux_armv7 and linux_arm64

scakemyer committed:
force exit with 0 code for Windows, fixes #214

scakemyer committed:
Update to v0.9.24

scakemyer committed:
disable background handling by default

scakemyer committed:
fix for raspberry after enabling linux_armv7 and linux_arm64, #331

scakemyer committed:
use armv7 for both rpi2/3, #331

scakemyer committed:
Update to v0.9.25

scakemyer committed:
add share_ratio_limit, seed_time_ratio_limit and seed_time_limit options, closes #147
add disable_dht option, #262, closes #38
move Custom provider timeout to Providers tab

scakemyer committed:
disable sorting by name for tvshows, fixes #120

scakemyer committed:
Results per page instead of Enable pagination, #279 #313

scakemyer committed:
add Update library command in maintenance settings, #107

scakemyer committed:
separate Add to library and Remove from library strings, #259

scakemyer committed:
add option for adding Specials to library, closes #104

scakemyer committed:
Update to v0.9.26

scakemyer committed:
add Trakt settings and strings, #31

add Dialog and Dialog_Config RPC methods

KillerJoeBR committed:
Updated Portuguese (Brazil) translation

Added new strings to the translation

scakemyer committed:
Merge pull request #362 from KillerJoeBR/patch-8

Updated Portuguese (Brazil) translation

zorg1977 committed:
Updated french translations

Some sentences are not translated due to a lack of understanding:
- "Seed time ratio limit (percent)" => how is this supposed to work? Seed time limit is ok for me, as well as Share ratio limit, but I don't get this one.
- "Add Specials to library" => what is "Specials"? I just didn't translated the word and kept it as is for now.

scakemyer committed:
add manual authorize action in advanced settings

assapir committed:
Update Hebrew string to 0.9.26

m4xc4v413r4 committed:
Updated Portuguese Translation

Added new strings, changed some other after looking at @vascool pull request #315.
I didn't copy everything he did since I didn't agree with some stuff, and some others I just changed with basis on his change but overall his suggestions were good.

I would love for @vascool to see if he can come up with anything for "seeders" and "watchlist" since I really have no idea, I never used another word for it and literal translations are pretty shitty. Most specifically, 30051 to 30053 and 30254 to 30256.

scakemyer committed:
Merge pull request #368 from meijin007/master

Update Hebrew string to 0.9.26

scakemyer committed:
Merge pull request #370 from m4xc4v413r4/patch-1

Updated Portuguese Translation

scakemyer committed:
Merge pull request #365 from zorg1977/patch-7

Updated french translations

Commit:
Updated German Translation

Updated German Translation

scakemyer committed:
Merge pull request #373 from eidermar/master

Updated German Translation

zorg1977 committed:
French translation updated

Changed "résolution" into "définition" as it should be the right term to use speaking about number of pixels (and not pixels density, which is what a resolution is)
Added a few translation left as-is in the previous commit

scakemyer committed:
Merge pull request #375 from zorg1977/patch-8

French translation updated

m4xc4v413r4 committed:
Updated Portuguese Translation v2

Missed a string that was changed on another update.

scakemyer committed:
Merge pull request #377 from m4xc4v413r4/patch-2

Updated Portuguese Translation v2

KillerJoeBR committed:
Update to Portuguese (Brazil) translation

New Trakt strings in the translation

scakemyer committed:
Merge pull request #379 from KillerJoeBR/patch-9

Update to Portuguese (Brazil) translation

scakemyer committed:
add Trakt collection strings, #31

scakemyer committed:
Update to v0.9.27

scakemyer committed:
Update to v0.9.28

KillerJoeBR committed:
Updated Portuguese (Brazil) translation

New trakt strings

scakemyer committed:
Merge pull request #384 from KillerJoeBR/patch-10

Updated Portuguese (Brazil) translation

scakemyer committed:
Update to v0.9.29

m4xc4v413r4 committed:
Updated Portuguese Translation

Added the new Trakt Collection strings.
Translation on par with 0.9.29

scakemyer committed:
Merge pull request #386 from m4xc4v413r4/patch-4

Updated Portuguese Translation

scakemyer committed:
add CloudHole API key setting, #387

move up clear cache and update actions

assapir committed:
Update Hebrew

scakemyer committed:
Merge pull request #390 from meijin007/master

Update Hebrew

Greenbaburu committed:
Update italian translation #30058 / 30113 / 30259

scakemyer committed:
Merge pull request #391 from Greenbaburu/patch-10

Update italian translation #30058 / 30113 / 30259

scakemyer committed:
Update to v0.9.30

scakemyer committed:
add Scrobble setting, #297

Filosofos committed:
Updated Greek translation 0.9.30

scakemyer committed:
Merge pull request #396 from Filosofos/patch-8

Updated Greek translation 0.9.30

penzinho committed:
Croatian translation updated up to v.0.9.30

zorg1977 committed:
French translation updated with new Trakt options

Added new Trakt options
Clean old Trakt entries (no more Trakt: prefix since everything is now under a Trakt menu)

scakemyer committed:
add Player_WatchTimes RPC method

scakemyer committed:
Merge pull request #399 from zorg1977/patch-9

French translation updated with new Trakt options

scakemyer committed:
Merge pull request #397 from penzinho/master

Croatian translation updated up to v.0.9.30

scakemyer committed:
add setting for library update delay on startup

scakemyer committed:
Update to v0.9.31

scakemyer committed:
cast library_update_delay to integer...

scakemyer committed:
Update to v0.9.32

m4xc4v413r4 committed:
Updated Portuguese Translation

Added new strings.
Updated as of v0.9.32

scakemyer committed:
Merge pull request #402 from m4xc4v413r4/patch-1

Updated Portuguese Translation

KillerJoeBR committed:
Updated Portuguese (Brazil) translation

Added new string to portuguese brazil translation

scakemyer committed:
Merge pull request #405 from KillerJoeBR/patch-11

Updated Portuguese (Brazil) translation

assapir committed:
Update Hebrew strings

scakemyer committed:
Merge pull request #408 from meijin007/master

Update Hebrew strings

scakemyer committed:
add string for previously chosen torrent, #148

scakemyer committed:
add search history strings, #411

m4xc4v413r4 committed:
Updated Portuguese Translation

Added new "Search History" strings

scakemyer committed:
Merge pull request #413 from m4xc4v413r4/patch-2

Updated Portuguese Translation

scakemyer committed:
Update to v0.9.33

KillerJoeBR committed:
Updated Portuguese (Brazil) translation

Added new strings to PT BR translation

scakemyer committed:
Merge pull request #417 from KillerJoeBR/patch-12

Updated Portuguese (Brazil) translation

assapir committed:
Update Hebrew for 0.9.33

scakemyer committed:
Merge pull request #420 from meijin007/master

Update Hebrew for 0.9.33

Commit:
German Translation Update

German updated to 0.9.33

scakemyer committed:
Merge pull request #427 from eidermar/master

German Translation Update

penzinho committed:
Croatian translation brought upt to speed

guidosarducci committed:
Better daemonize quasar binary by closing non-std inherited files.
Fixes problem updating skin addons (#41). Updates both Unix and Windows.

jellespijker committed:
Dutch translations updated

scakemyer committed:
Merge pull request #448 from penzinho/master

Croatian translation brought upt to speed

zorg1977 committed:
Update french strings.po

Missing sentences added

scakemyer committed:
Merge pull request #451 from peer23peer/master

Dutch translations updated

scakemyer committed:
Merge pull request #474 from zorg1977/patch-10

Update french strings.po

scakemyer committed:
Merge pull request #469 from guidosarducci/fix-close-quasar-fds

Greenbaburu committed:
Update italian translation #30059 / 30113 / 30262

scakemyer committed:
Merge pull request #488 from Greenbaburu/patch-11

Update italian translation #30059 / 30113 / 30262

scakemyer committed:
Update to v0.9.34

achaw committed:
Update Spanish Translation

scakemyer committed:
Merge pull request #505 from achaw/patch-1

Update Spanish Translation

scakemyer committed:
silence flake8 being picky

scakemyer committed:
add gitter badge

scakemyer committed:
Update to v0.9.35

maxmousee commented:
I would change "Ativar (use à sua responsabilidade)" por "Ativar (use por sua conta e risco)".
But this is not wrong, I just think it would sound more natural.
Excellent translation!

scakemyer committed:
Update to v0.9.36

scakemyer committed:
fix with_cookies method for providers, #376 #38

scakemyer committed:
fix E305 linting..

scakemyer committed:
add use_cloudhole setting, update codec indices

scakemyer committed:
set IMDBNumber infolabel #566

scakemyer committed:
add translation for Lists

scakemyer committed:
Update to v0.9.37

scakemyer committed:
new Krypton theme for stream selection dialog

scakemyer committed:
checkout 20 commits from quasar-binaries for Travis

scakemyer committed:
Update to v0.9.38

scakemyer committed:
add library update frequency setting, closes #464
add setting for sync frequency of Trakt lists to library, #398

scakemyer committed:
add separate action to reset clearances

scakemyer committed:
finish Trakt lists syncing

scakemyer committed:
Update to v0.9.39

scakemyer committed:
pass payload as exception message, translation for Item already in library

scakemyer committed:
add Ignore duplicates setting #32, update Trakt user lists #398

scakemyer committed:
Update to v0.9.40

Lam4hound committed:
Finnish translation (#583)

Finnish translation

scakemyer committed:
Update to v0.9.41

scakemyer committed:
set Mediatype and Code, from DBTYPE and IMDBNumber, closes #566

scakemyer committed:
add 'Ask to keep' setting and strings #14

scakemyer committed:
add TV scraper setting for duplicate handling #32

scakemyer committed:
Update to v0.9.42

scakemyer committed:
add GetAllSettings method to RPC server

scakemyer committed:
add Disable UPNP & NATPMP setting, #385

scakemyer committed:
add 'Toggle watched' string #24

scakemyer committed:
Update to v0.9.43

assapir committed:
update Hebrew to latest (#592)

scakemyer committed:
add OpenSubtitles settings for user/pass

scakemyer committed:
get Year infolabel...

scakemyer committed:
set a bunch of missing infolabels #256

scakemyer committed:
Update to v0.9.44

m4xc4v413r4 committed:
Updated Portuguese Translation (#593)

Couple changes to previous strings, added all the new strings so it's all up to date as of today.

johnnyslt committed:
Updated Romanian Translation (#596)

scakemyer committed:
remove Cast and CastAndRole infolabels, closes #597

scakemyer committed:
Update to v0.9.45

KillerJoeBR committed:
Updated Portuguese (Brazil) translation (#600)

Added new strings to the brazilian portuguese translation

Albertron86 commented:
Hi. I tested the plugin on a vu + decoder with enigma2, and it seems that it is not supported. Is it possible to add mipsel support to run Quasar in enigma2 decoder? Best regards.

scakemyer committed:
Update to v0.9.46

scakemyer committed:
add web UI

scakemyer committed:
Update to v0.9.47

scakemyer committed:
update web UI to use new /playuri endpoint

scakemyer committed:
Update to v0.9.48

scakemyer committed:
add encryption policy and proxy type settings

scakemyer committed:
add play button in web UI and update favicon

scakemyer committed:
add user agent spoofing setting (inadvisable)

scakemyer committed:
no need to clear caches when settings change, closes #616

scakemyer committed:
set a more reasonable listen port range

scakemyer committed:
Update to v0.9.49

scakemyer committed:
add contributing, issue and pull request templates

scakemyer committed:
add tuned storage setting

scakemyer committed:
change 'Ask to keep' to 'Ask to delete'

scakemyer committed:
Update to v0.9.50

scakemyer committed:
web UI button to add torrent without streaming

scakemyer committed:
Update to v0.9.51

scakemyer committed:
fix double output logging for invoked providers

scakemyer committed:
Update to v0.9.52

Greenbaburu committed:
Italian Translation Update to v0.9.52 (#630)

Italian Translation Update to v0.9.52

assapir committed:
Update Hebrew (#629)

scakemyer committed:
add 'Match found in your active torrents' string

scakemyer committed:
callback response code as debug log..

scakemyer committed:
Update to v0.9.53

scakemyer committed:
Update to v0.9.54

m4xc4v413r4 committed:
Updated Portuguese Translation (#639)

Added strings to get up-to-date with v0.9.54

scakemyer committed:
mark overlay status offset as subsetting

gitignore those big .js.map files

Lam4hound committed:
Update for Finnish translations (#640)

Just translated all the new strings.

scakemyer committed:
uncheck item in web UI after deleting

scakemyer committed:
Update to v0.9.55

scakemyer committed:
add .gitlab-ci.yml

assapir committed:
Hebrew string update (#644)

assapir committed:
Remove redundant "pre-release" option from setting (#646)

scakemyer committed:
clean up addon.xml, list included languages

scakemyer committed:
clean up redundant english strings, add Burst install strings

scakemyer committed:
add Enable all/Disable all strings, #493

scakemyer committed:
Update to v0.9.56

assapir committed:
Update hebrew strings (#650)

scakemyer committed:
add 'Delete torrent and files' context menu, closes #423

scakemyer committed:
add toggle to delete files with torrent in web UI

scakemyer committed:
add dimmer in web UI for error messages

scakemyer committed:
add configurable initial buffering timeout, closes #253

scakemyer committed:
Update to v0.9.57

scakemyer committed:
link web UI header to self, add Made with love mention

scakemyer committed:
Update to v0.9.58

johnnyslt committed:
Updated Romanian Translation (#658)

m4xc4v413r4 committed:
Updated Portuguese Translation (#660)

Added new strings.
Up-to-date as of v0.9.58

assapir committed:
Update Hebrew translation (#663)

scakemyer committed:
add search to web UI

scakemyer committed:
Update to v0.9.59

scakemyer committed:
hotfix for web UI using wrong host, #659

scakemyer committed:
Update to v0.9.60
